HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -2022,6 +2023,7 @@ def self.process_heat_pump_adjustment(hvac_sizing_values, weather, hvac_heating,
       heating_db = @hpxml.header.manualj_heating_design_temp
     end
 
+    # FIXME: Replace performance coefficients with new defaulting correlations
```

Comment:
There's a sequence issue to replace coefficients with the new performance data,. The performance data defaulting method has to be called after the hvac sizing because we need the autosized capacity as 47F rated capacity, and then build other data points based on that, but here the sizing needs the detailed performance data to adjust the capacity. So the correlation cannot be directly called. Is it acceptable to retain the coefficients here? Or we can write the normalized correlation (get rid of capacity) here based on the defaulting method.

Reply:
I think I understand. We are using a linear heating capacity as a function of temperature relationship going forward, is that right? Can you just use the heating capacity retention fraction -- either A) entered directly by the user, B) calculated from detailed performance data, or C) defaulted?

Reply:
Oh, I take that back -- the detailed performance data may not be linear.

Reply:
So if the detailed performance data is provided, you can use that. If it's not provided, you can use the heating capacity retention fraction (user entered or defaulted). Does that work? I realize that's two pathways for the calculations, and that you were perhaps hoping for a single pathway.

Reply:
If the detailed performance data is provided, the autosizing shouldn't mean too much since we plan to require the rated capacity. So that won't be an issue for the simulation. But if we report the sizing for other uses then yes we should use the user inputs to adjust. 
And for path when detailed performance data is not provided, then the heating capacity retention path sounds like the most ideal one, and cooling we can apply Jon's defaulting approach. In that case, the only independent variable is the outdoor temperature, the code to introduce indoor condition sensitivity is [here](https://github.com/NREL/OpenStudio-HPXML/pull/1317/files#diff-1747bd7cd2671c2c55032c376b1d67f9d6a37d6df45e254574430f5796c5b830R2871). It's introduced by applying 1 speed cutler coefficients. The capacity coefficients of the indoor condition terms (second and third terms of cap_ft_spec), though, are not nothing, but look different across different systems: 
single speed: 

> cap_ft_spec = [[3.68637657, -0.098352478, 0.000956357, 0.005838141, -0.0000127, -0.000131702]]

var speed ashp max speed: 

> cap_ft_spec = [[1.8769221728395058, -0.04768641975308643, 0.0006885802469135801, 0.006643395061728395, 1.4209876543209876e-05, -0.00024043209876543206]]

mini-split max speed:

> cap_ft_spec = [[0.7531983499655835, 0.003618193903031667, 0.0, 0.006574385031351544, -6.87181191015432e-05, 0.0]]

So I am not sure if we want to introduce this complexity here if the 1 speed correction approach is something we may revisit in the future.